agent/structs/intention.go

```diff
@@ -26,6 +26,8 @@ const (
 	// fix up all the places where this was used with the proper namespace
 	// value.
 	IntentionDefaultNamespace = "default"
+
+	MaxIntentionPrecedence = 9
```

Comment:
Did you mean to use this instead of the wildcard check?

Reply:
Originally, yeah, but then remembered config intentions don't have a precedence.

Reply:
Removed this.